schism/compiler.ss

```diff
@@ -607,6 +682,21 @@
         (trace-and-error name 'lookup "unbound identifier"))
       ((cdr pair))))
 
+  (define (make-export-environment lib)
+    (let ((name (cadr lib))
+          (body (cddr lib)))
+      (let
+          ((exports (cdar body))
+           (defs (cddr body)))
```

Comment:
Perhaps put all these in a let* ? As you wish :)

Reply:
Done.

I'm always conflicted in cases like this between readability and not wanting to impose ordering constraints where they aren't needed.